educator.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'person'
+
+class Educator < Person
+
+  attr_reader  :salary, :target_raise
+
+  def initialize(options={})
+  	super
+    @phase = 3
+  end
+
+  def set_phase(num)
+    @phase = num
+    "Cool, I've always wanted to teach phase #{num}!"
+  end
+
+  def salary=(new_salary)
+    puts "This better be good!"
+    @salary = new_salary
+  end
+
+  def receive_raise(raise)
+    @salary += raise
+  end
+
+  def set_performance_rating(rating)
+    response = ""
+    if rating > self.class::RATING_REQUIREMENT
```

Comment:
Yep, that's the standard way to access a constant.  The thing I mentioned that you should not do is check your own type and condition off of it.  This is just accessing a constant and using it instead of hard-coding.

Think of it this way: if you added a new subclass of Educator, would your code force you to change the code in Educator?  If no (as in this case, where all you'd need to do is specify the constant in the subclass), you're fine.  If yes (as in the "am I this type?" sort of check you were doing in your module, where you'd need to add another check for the new type), you should avoid it.